### PR TITLE
Move the project to the Hibernate Search codebase

### PR DESCRIPTION
**Pull request by yrodiere**
This should make it clear that no contribution is expected on this project, but that it still lives elsewhere.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,5 +26,3 @@ control (start/stop/restart), logging, checkpointing and parallelization.
 For more information, please check http://mincong-h.github.io/gsoc-hsearch
```
Comment (mincong-h) [inline README.md]: Could your please remove this line? The documentation is obsolete, so it's better to remove it too. I'll also disable the graphic branch `gh-pages`.

**Review by mincong-h — COMMENTED** [review]
Thanks for your PR, @yrodiere.

File: README.md (hunk 1 above)
Comment (yrodiere) [reply README.md]: Right. I removed it.

**Comment by mincong-h** [comment]
Merged. Thanks @yrodiere!

**Pull request merged**